src/features/modalActionMenusVisibilitySlice.js

```diff
@@ -7,6 +7,18 @@ const initialState = {
     copyMenuOpen: false,
     moveMenuOpen: false,
     deleteMenuOpen: false,
+    attachmentMenuOpen: {
+      status: false,
+      id: 0
```

Comment:
What's the significance of id?

Reply:
There are two buttons that can open the attachmentMenu. By setting the ID we are able to show one of the menus rather than both.

Reply:
Fair enough